src/pages/Clients/Clients.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../styles/variables" as *;
+@use "../../styles/main.scss" as *;
+
+.clients-container {
+  background-color: #ebebf5;
+  height: 100vh;
```

Comment:
Can you take a look at this please i don't think it needs it

Reply:
We need the backgroud color for the page otherwise, nothing applied to it. We can remove height as it's not needed.